src/app/home/modules/schedule/components/schedule-item-detail/schedule-item-detail.component.scss

```diff
@@ -1,20 +1,17 @@
 .schedule-item-detail {
-	h1,
 	h3 {
-		font-weight: 100;
+		font-weight: 400;
 	}
 
 	section {
-		padding-top: 30px;
```

Comment:
The content feels very packed without padding though 🤔 . I agree `30` was a bit much, but I feel like we might want at least 10? I also had to add `margin: 0` to the `h3` tag on line 2 because the margin was doing something weird with the padding.

Anyways, feel free to ignore 😂 .

With no padding
![image](https://user-images.githubusercontent.com/1913805/223653783-f65e8ad9-2ebd-4a33-b52a-5cb0f7f6e709.png)

With 10px padding
![image](https://user-images.githubusercontent.com/1913805/223654025-8510af39-2145-4881-ab1e-59b0f307a5fb.png)